src/tests/utils.ts

```diff
@@ -41,23 +41,23 @@ export async function queryNamespaces(
   return [...orgNamespaces, ...installedNamespaces];
 }
 
-export const queryAll = async <T>(
+export const queryAll = async <R>(
   connection: Connection,
   query: string,
   tooling = false
-): Promise<QueryResult<T>> => {
+): Promise<QueryResult<R>> => {
   const conn = tooling ? connection.tooling : connection;
-  const allRecords: T[] = [];
-  let result = await conn.query<T>(query);
+  const allRecords: R[] = [];
+  let result = await conn.query<R>(query);
   allRecords.push(...result.records);
   while (!result.done) {
-    result = (await conn.queryMore(result.nextRecordsUrl)) as QueryResult<T>;
+    result = (await conn.queryMore(result.nextRecordsUrl)) as QueryResult<R>;
     allRecords.push(...result.records);
   }
 
   return {
     done: true,
     totalSize: allRecords.length,
     records: allRecords
-  } as QueryResult<T>;
+  } as QueryResult<R>;
```

Comment:
@daphne-sfdc I removed the assertion w/o complaint from elsint or others.

See if the same holds for you

Reply:
Yes you're right - I removed the assertion and the tests still all pass.  The functionality also continues to work as expected.  Thanks for catching that!